Autopilot/Inc/gps.hpp

```diff
@@ -49,51 +58,74 @@ class Gps
         virtual void GetResult(GpsData_t *Data) = 0;
 };
 
-class NEOM8 : public Gps 
+class NEOM8 : public Gps
 {
 	public:
 		NEOM8(const NEOM8*) = delete;
 		static NEOM8* GetInstance();
 
 		/**
-         * Triggers interrupt for new GPS measurement - stores raw data in variables and returns right away
-         * */
+		 * Triggers interrupt for new GPS measurement - stores raw data in variables and returns right away
+		 * */
 		void BeginMeasuring() {}; //No need to call this to get data. Just call GetResult()
```

Comment:
you can actually just get rid of this. But a quick not somewhere that this gps just spews data without anyone asking would be good.